### PR TITLE
ufw: add Spanish translation

### PR DESCRIPTION
**Pull request by maegop**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/ufw.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# ufw
+
+> Cortafuegos sin complicaciones (Uncomplicated Firewall).
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:3]:
```suggestion
> Cortafuegos sin complicaciones (_Uncomplicated Firewall_).
```

File: pages.es/linux/ufw.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+> Interfaz de usuario de iptables para facilitar la configuración de un firewall.
+> Más información: <https://wiki.ubuntu.com/UncomplicatedFirewall>.
+
+- Activar ufw:
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:7]:
```suggestion
- Activa ufw:
```

File: pages.es/linux/ufw.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`ufw enable`
+
+- Desactivar ufw:
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:11]:
```suggestion
- Desactiva ufw:
```

File: pages.es/linux/ufw.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+`ufw disable`
+
+- Mostrar reglas del ufw, junto con sus números:
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:15]:
```suggestion
- Muestra reglas del ufw, junto con sus números:
```

File: pages.es/linux/ufw.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+`ufw status numbered`
+
+- Permitir el tráfico entrante en el puerto 5432 en este host con un comentario que identifique el servicio:
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:19]:
```suggestion
- Permite el tráfico entrante en el puerto 5432 en este host con un comentario que identifique el servicio:
```

File: pages.es/linux/ufw.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`ufw allow {{5432}} comment "{{servicio}}"`
+
+- Permitir sólo el tráfico TCP desde 192.168.0.4 a cualquier dirección de este host, en el puerto 22:
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:23]:
```suggestion
- Permite sólo el tráfico TCP desde 192.168.0.4 a cualquier dirección de este host, en el puerto 22:
```

File: pages.es/linux/ufw.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+`ufw allow proto {{tcp}} from {{192.168.0.4}} to {{cualquier}} port {{22}}`
+
+- Denegar tráficoen el puerto 80 en este host:
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:27]:
```suggestion
- Deniega tráfico en el puerto 80 en este host:
```

File: pages.es/linux/ufw.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+`ufw status numbered`
+
+- Permitir el tráfico entrante en el puerto 5432 en este host con un comentario que identifique el servicio:
+
+`ufw allow {{5432}} comment "{{servicio}}"`
+
+- Permitir sólo el tráfico TCP desde 192.168.0.4 a cualquier dirección de este host, en el puerto 22:
+
+`ufw allow proto {{tcp}} from {{192.168.0.4}} to {{cualquier}} port {{22}}`
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:25]: The `any` command shouldn't be translated.
```suggestion
`ufw allow proto {{tcp}} from {{192.168.0.4}} to {{any}} port {{22}}`
```

File: pages.es/linux/ufw.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+- Denegar tráficoen el puerto 80 en este host:
+
+`ufw deny {{80}}`
+
+- Denegar todo el tráfico al puerto 22:
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:31]:
```suggestion
- Deniega todo el tráfico al puerto 22:
```

File: pages.es/linux/ufw.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+`ufw deny proto {{udp}} from {{cualquier}} to {{cualquier}} port {{22}}`
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:33]:
```suggestion
`ufw deny proto {{udp}} from {{any}} to {{any}} port {{22}}`
```

File: pages.es/linux/ufw.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+- Eliminar una regla concreta. El número de la regla puede obtenerse del comando `ufw status numbered`:
```
Comment (navarroaxel) [inline pages.es/linux/ufw.md:35]:
```suggestion
- Elimina una regla concreta. El número de la regla puede obtenerse del comando `ufw status numbered`:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages.es/linux/ufw.md (hunk 1 above)
Comment (marchersimon) [reply pages.es/linux/ufw.md]: Why the underscores?

Comment (navarroaxel) [reply pages.es/linux/ufw.md]: Because these are an English words. it's a common usage.

Comment (marchersimon) [reply pages.es/linux/ufw.md]: Is this indented to be italic? Because I don't think most clients support that.

Comment (navarroaxel) [reply pages.es/linux/ufw.md]: If it's not supported the words are still readable. pages like `supervisorctl` and `csvcut` have `**` and we have no issues with them.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**